databuilder/models/table_metadata.py

```diff
@@ -191,10 +199,16 @@ def create_next_node(self):
 
     def _create_next_node(self):
         # type: () -> Iterator[Any]
-        yield {NODE_LABEL: TableMetadata.TABLE_NODE_LABEL,
-               NODE_KEY: self._get_table_key(),
-               TableMetadata.TABLE_NAME: self.name,
-               TableMetadata.IS_VIEW: self.is_view}
+
+        table_node = {NODE_LABEL: TableMetadata.TABLE_NODE_LABEL,
+                      NODE_KEY: self._get_table_key(),
+                      TableMetadata.TABLE_NAME: self.name,
+                      TableMetadata.IS_VIEW: self.is_view}
+        if self.attrs:
+            for k, v in self.attrs.items():
+                if k not in table_node:
```

Comment:
Should we make it fail if it already exists?

Reply:
sync offline. The __init__ will fail if there are duplicate attributes.